### ISSUE TITLE
Are you planning on further development or is this about all you want from it?

### ISSUE THREAD

**Issue by Charuru** [jochemstoel/Procyon#1]

**Comment by jochemstoel** [comment]
Yes I definitely intend to further develop this, especially if it turns out there is a demand for it.
I created it initially because I needed it myself and the answers on the Raccoon GitHub issues were not very satisfying but I like to hear from others.

**Comment by Charuru** [comment]
Any roadmap?
